src/daemon/gateway.go

```diff
@@ -18,17 +19,22 @@ import (
 	"github.com/skycoin/skycoin/src/visor/historydb"
 )
 
+const (
```

Comment:
This should be defined in api/http.go with the others

Reply:
done